Removing jsonpath-plus from pubrepo

vaibhav-ozone opened the pull request:
jsonpath-plus has been causing a few compatibility issues and so we've replaced it with the native
javascript code.

PratapTechSavvy reviewed (commented):
there is another reference of `jsonpath-plus` in `pubrepo/ozone-jwks/package.json`

vaibhav-ozone commented:
> there is another reference of `jsonpath-plus` in `pubrepo/ozone-jwks/package.json`

Yes. That was also removed. But for some reason, it didn't make it into this commit. Added now.
